.github/workflows/build_and_test.yml

```diff
@@ -32,11 +32,12 @@ jobs:
       style_checker_image_name: "ghcr.io/citusdata/stylechecker"
       style_checker_tools_version: "0.8.18"
       sql_snapshot_pg_version: "17.5"
-      image_suffix: "-dev-d28f316"
+      image_suffix: "-dev-aa7482a"
```

Comment:
So, this image suffix is referring to this commit https://github.com/citusdata/the-process/commit/aa7482a
In this commit, 18beta2 images are not present.

Reply:
When beta2 packages are published, I will generate test images.